server/src/main/java/org/elasticsearch/rest/action/admin/cluster/RestNodesCapabilitiesAction.java

```diff
@@ -54,16 +54,26 @@ protected RestChannelConsumer prepareRequest(RestRequest request, NodeClient cli
             ? new NodesCapabilitiesRequest(client.getLocalNodeId())
             : new NodesCapabilitiesRequest();
 
+        // Handle the 'path' parameter safely, assign an empty string if null
+        String path = request.param("path");
+        if (path != null) {
```

Comment:
Instead of checking for null, we can just use the default value parameter:

    .path(request.param("path", "/"))

Reply:
Hi @thecoop,

I've made the requested change to use request.param("path", "/") as the default value for the path parameter instead of checking for null. Please let me know if there are any further adjustments needed.

Thank you!